### PR TITLE
Add logging to bluegreen-pipeline.yaml to help diagnose flake

### PR DESCRIPTION
**Pull request by jim-minter**
Also rewrite blue-green pipeline extended test to avoid hang on error

@openshift/devex

### CODE REVIEW

File: test/extended/builds/pipeline.go [hunk 1]
```diff
@@ -100,6 +95,13 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 	})
 
 	g.Context("Pipeline with maven slave", func() {
+		g.AfterEach(func() {
+			if os.Getenv(jenkins.DisableJenkinsGCStats) == "" {
+				g.By("stopping jenkins gc tracking")
+				ticker.Stop()
+			}
+		})
+
```
Comment (gabemontero) [inline test/extended/builds/pipeline.go]: This review comment is just a question for my enlightenment.  The move of the `g.AfterEach` is interesting.  I'm curious as to how exactly moving within the ginkgo context helped with exiting the test on the failure.

thanks

Comment (jim-minter) [inline test/extended/builds/pipeline.go]: It's more about preventing a page of errors at the end of the each test. Ginkgo AfterEaches run in scope order, inner to outer.  By pushing this AfterEach in a scope, we ensure that it runs before the framework deletes the namespace, not after.

Comment (gabemontero) [inline test/extended/builds/pipeline.go]: Interesting ... yes, that sounds like a good general practice to at least consider across our extended tests in general.

Thanks for the clarification.

File: examples/jenkins/pipeline/bluegreen-pipeline.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ objects:
             }
 
             stage("Go Live") {
-              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0"
+              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0 --loglevel=4"
```
Comment (bparees) [inline examples/jenkins/pipeline/bluegreen-pipeline.yaml]: this is all jenkins job output, are we actually getting and displaying the jenkins job output somewhere in the test case?

Comment (bparees) [inline examples/jenkins/pipeline/bluegreen-pipeline.yaml]: @jim-minter bump (otherwise this PR lgtm)

Comment (jim-minter) [inline examples/jenkins/pipeline/bluegreen-pipeline.yaml]: ah - the answer is yes (in `func debugAnyJenkinsFailure`).

[I tried replying via e-mail from my mobile to GitHub yesterday evening, and clearly it didn't work!]

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**